src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -225,6 +269,13 @@ public void apply(final VariantContext vc, final ReadsContext readsContext, fina
         }
     }
 
+    private void ValidateGVCFVariant(final VariantContext vc) {
+        if (!vc.hasAllele(GATKVCFConstants.NON_REF_SYMBOLIC_ALLELE)) {
+            throw new TribbleException.InternalCodecException(String.format("In a GVCF all records must contain a %s allele. Offending record: %s",
```

Comment:
This doesn't seem like the right exception to throw.  It seems like the others throw a UserException if validation fails, shouldn't this match?  Should it respect the warning only mode that other validations seem to handle?